### ISSUE TITLE
Delegate registration transaction doesn't pass validation on the node side.

### ISSUE THREAD

**Issue by alepop** [LiskHQ/lisk-js#614]
### Description
Delegate registration transaction doesn't pass validation on the node side.
When you create a delegate registration transaction with `lisk-js 1.0.0` it creates transaction object with `recipientId` field equal to `null`. This transaction doesn't pass the validation on the `lisk 1.0.0` node side when you pass it through the api. And return the error: 
```
{
	"message": "Validation errors",
	"errors": [{
		"code": "INVALID_REQUEST_PARAMETER",
		"message": "Invalid parameter (transaction): Value failed JSON Schema validation",
		"in": "body",
		"name": "transaction",
		"errors": [{
			"code": "INVALID_TYPE",
			"message": "Expected type string but found type null",
			"path": ["recipientId"]
		}, {
			"code": "INVALID_FORMAT",
			"message": "Object didn't pass validation for format address: null",
			"path": ["recipientId"]
		}]
	}]
}
```

### Steps to Reproduce

1. Create a transaction for delegate registration with `lisk-js 1.0.0`
2. Pass it to the node `lisk 1.0.0` through the `api` (api/transactions)

**Expected behavior:** 
A delegate registration transaction will pass validation step

**Actual behavior:**
It causes a validation error

**Reproduces how often:**
Constatly

### Versions
`lisk-js 1.0.0`
`lisk 1.0.0`

**Comment by alepop** [comment]
Also, registering the second passphrase and multi-signature transaction have the same behavior for signed tx and don't pass validation step on the core.

**Comment by shuse2** [comment]
default of `recipientId` is `null` on `lisk-js`, but according to the API docs, on `core API`, `recipientId` is a required string field.
Therefore, the `recipientId` should be default to empty string on `lisk-js` too.

**Comment by alepop** [comment]
@shuse2 yes, you are right, with an empty string value, transaction pass validation step.
